tachio/src/vision.js

```diff
@@ -11,7 +11,7 @@ const fetchImageDescription = async () => {
   logger.info(`Requesting description of image at ${imageUrl}`);
```

Comment:
@kyphan1 is this `{imageUrl}` configured correctly to accept the ephemeral URL from Missive in the webhook payload of an uploaded attachment?

Reply:
Yes, It's set in:
https://github.com/PublicDataWorks/tachio/blob/main/tachio/api.js#L170
https://github.com/PublicDataWorks/tachio/blob/main/tachio/src/vision.js#L46

btw, their usage of global variables is an anti-pattern. I will refactor it soon.